src/client/index.js

```diff
@@ -269,73 +295,78 @@ export class BncClient {
   /**
    * Create and sign a multi send tx
    * @param {String} fromAddress
-   * @param {Array} transfers
+   * @param {Array} outputs
    * @example
-   * const transfers = [
+   * const outputs = [
    * {
    *   "to": "tbnb1p4kpnj5qz5spsaf0d2555h6ctngse0me5q57qe",
    *   "coins": [{
    *     "denom": "BNB",
    *     "amount": 10
-   *   }]
+   *   },{
+    *     "denom": "BTC",
+    *     "amount": 10
+    *   }]
    * },
    * {
    *   "to": "tbnb1scjj8chhhp7lngdeflltzex22yaf9ep59ls4gk",
    *   "coins": [{
    *     "denom": "BTC",
    *     "amount": 10
-   *   }]
+   *   },{
+    *     "denom": "BNB",
+    *     "amount": 10
+    *   }]
    * }]
    * @param {String} memo optional memo
    * @param {Number} sequence optional sequence
    * @return {Promise} resolves with response (success or fail)
    */
-  async multiSend(fromAddress, transfers, memo="", sequence=null) {
+  async multiSend(fromAddress, outputs, memo="", sequence=null) {
     if(!fromAddress) {
       throw new Error("fromAddress should not be falsy")
     }
 
-    if(!Array.isArray(transfers)) {
-      throw new Error("transfers should be array")
+    if(!Array.isArray(outputs)) {
+      throw new Error("outputs should be array")
     }
 
-    checkTransfers(transfers)
+    checkOutputs(outputs)
 
-    transfers.forEach(item=>{
+    //sort denom by alphbet and init amount
+    outputs.forEach(item=>{
+      item.coins = item.coins.sort((a,b)=> a.denom.localeCompare(b.denom))
       item.coins.forEach(coin=>{
-        coin.amount = Number(coin.amount * Math.pow(10, 8))
+        const amount = new Big(coin.amount)
+        coin.amount = Number(amount.mul(Math.pow(10, 8)).toString())
       })
     })
 
     const fromAddrCode = crypto.decodeAddress(fromAddress)
     const inputs = [{ address: fromAddrCode, coins: [] }]
```

Comment:
if move this into  `calInputCoins`, it will build the inputs entirely within it. but i also have to multily `Math.pow(10,8) ` in outputs,thus will add more extra code. So i init the amount multiply `Math.pow(10,8) `  at the beginning in `multiSend`

Reply:
Replicating code is fine if it makes more sense to e.g. build the inputs in the method